### DISCUSSION TITLE
How to edit lnd.conf options when running btcpayserver docker install with LND service?

### DISCUSSION THREAD

**Discussion by wroscoe** [btcpayserver/btcpayserver#2651] [Technical Support] [answered]
I'd like to increase the minimum channel size that others can open to my LND node to 10M sats. To do this I need to change the `minchansize` option in lnd.conf and restart lnd. Given that the the docker-compose and volumes are mostly generated by the btcpayserver scripts I don't know how to update the lnd.conf. 

If someone can share a good way to do this I'd happily create a PR to update the btcpayserver documentation.

Thanks.

**Comment by dennisreimann** [comment, marked as answer]
This was brought up multiple times recently, so I figured it would be good to add it to the docs :)

PR is btcpayserver/btcpayserver-doc#942 -- review welcome, let me know if something is unclear.

**Reply by wroscoe** [reply]
Thanks for this. Worked perfectly.
